src/app/services/task.service.ts

```diff
@@ -44,6 +44,7 @@ import { GoogleGenerativeAIFetchError } from '@google/generative-ai';
 import { v4 as uuidv4 } from 'uuid';
 import { GoogleGenerativeAI } from '@google/generative-ai';
 import { environment } from '../../environments/environments';
+// TODO: import Vertex AI
```

Comment:
In addition to adding a comment line here, wdyt about also _commenting out_ this import statement [1] to make it clear that it's not needed until you decide to start using Vertex AI?

[1] `import { getVertexAI, getGenerativeModel } from 'firebase/vertexai-preview';`